src/applications/vaos/utils/selectors.js

```diff
@@ -71,10 +70,14 @@ export const vaosExpressCareNew = state =>
 export const selectFeatureToggleLoading = state => toggleValues(state).loading;
 const vaosFlatFacilityPage = state =>
   toggleValues(state).vaOnlineSchedulingFlatFacilityPage;
+const vaosFlatFacilityPageSacramento = state =>
+  toggleValues(state).vaOnlineSchedulingFlatFacilityPageSacramento;
```

Comment:
Is this flag added in other PRs? I don't think it's in the featureFlagNames.js file.

Reply:
Added to `featureFlagNames.js`.  Corresponding `vets-api` PR is here: https://github.com/department-of-veterans-affairs/vets-api/pull/5286.  Did I miss any others?  Haven't created a feature flag before.

Reply:
Nope, those are the two pieces.